migrator/migrations/n_43_to_n_44_postgres_report_configurations/legacy/rocksdb_plugin.go

```diff
@@ -0,0 +1,62 @@
+// Code generated by rocksdb-bindings generator. DO NOT EDIT.
+package legacy
+import (
+	"context"
+
+	"github.com/gogo/protobuf/proto"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/db"
+	"github.com/stackrox/rox/pkg/db/mapcache"
+	"github.com/stackrox/rox/pkg/rocksdb"
+	generic "github.com/stackrox/rox/pkg/rocksdb/crud"
+)
+
+var (
+	bucket = []byte("report_configs")
+)
+
+type Store interface {
+	UpsertMany(ctx context.Context, objs []*storage.ReportConfiguration) error
+	Walk(ctx context.Context, fn func(obj *storage.ReportConfiguration) error) error
+}
+
+type storeImpl struct {
+	crud db.Crud
+}
+
+func alloc() proto.Message {
+	return &storage.ReportConfiguration{}
+}
+
+func keyFunc(msg proto.Message) []byte {
+	return []byte(msg.(*storage.ReportConfiguration).GetId())
+}
+
+// New returns a new Store instance using the provided rocksdb instance.
+func New(db *rocksdb.RocksDB) (Store, error) {
+	baseCRUD := generic.NewCRUD(db, bucket, keyFunc, alloc, false)
+	cacheCRUD, err := mapcache.NewMapCache(baseCRUD, keyFunc)
```

Comment:
We don't want to have the mapcache in these instances

Reply:
Fixed.